hlxsites/maidenform
# [PWA] Base Integration with build process and component blocks

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #184 

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.page/
- After: 
    - Homepage: https://issue-184--maidenform--hlxsites.hlx.page (should not have performance impact)
    - Login: https://issue-184--maidenform--hlxsites.hlx.page/customer/account/login
    - Account Overview: https://issue-184--maidenform--hlxsites.hlx.page/customer/account
        - Order History: https://issue-184--maidenform--hlxsites.hlx.page/customer/order-history
        - Wishlist: https://issue-184--maidenform--hlxsites.hlx.page/customer/wishlist
        - Address book: https://issue-184--maidenform--hlxsites.hlx.page/customer/address
        - Account information: https://issue-184--maidenform--hlxsites.hlx.page/customer/account/edit
        - Stored payment methods: https://issue-184--maidenform--hlxsites.hlx.page/customer/cards
    - Cart: https://issue-184--maidenform--hlxsites.hlx.page/checkout/cart
    - Checkout: https://issue-184--maidenform--hlxsites.hlx.page/checkout


BEFORE MERGING
* Remove mock service worker
* Ensure that everything works on stage and that the stage GraphQL endpoint is used.
